### ISSUE TITLE
File created during upload

### ISSUE THREAD

**Issue by Jeggo65** [OllisGit/OctoPrint-DisplayLayerProgress#207]
Hello,

first of all the plugin is working :-)
When I upload a file to OctoPrint via Pursa Slicer 2.3 (e.g. local OctoPrint drive: Test/Test_Cube.gcode), a file in the root directory with the same name is created and contains this:

; BEGIN DISPLAYLAYERPROGRESS SETTINGS
; DisplayLayerProgress_layerIndicatorProcessed = true
; END   DISPLAYLAYERPROGRESS SETTINGS

In my setup (latest OctoPi and OctoPrint versions), the file remains in the root directory after the print is finished.
Could this file (automatically) be deleted after the print or even before if the gcode is scanned and processed?
Or if this is not possible could the file be generated in an extra directory (e.g. DLP)?

CU

Jens

**Comment by OllisGit** [comment]
Thx, for reporting!
I never tested the function to upload an gcode to an sub-folder.
I think I only used the filename instead of the full-path.

I will take a look on this.

**Comment by Jeggo65** [comment]
Maybe it is just a good idea to hide these files or put them in a folder, that is not shown in OctoPrint. When you select this file, you will get an error, that no layer information in this file. As far as I understand it, these files do not contain any data, that has to be edited by the user.

**Comment by HedgesHandmade** [comment]
I'm experiencing the same issue as it relates to the DeleteAfterPrinting plug-in. I have it set to move the file to the _archive folder after successfully printing, which it does successfully, but it also leaves the same 130 byte file with the same filename in the main file list. The text inside the file reads:

```
; BEGIN DISPLAYLAYERPROGRESS SETTINGS
; DisplayLayerProgress_layerIndicatorProcessed = true
; END   DISPLAYLAYERPROGRESS SETTINGS
```

**Comment by OllisGit** [comment]
Hi! 
Fixed in lates release V1.125.4

Please test and give feedback.

Thx, in advance!
Olli

**Comment by Jeggo65** [comment]
Thanx Olli,

it is working. Text lines are added to the original file.

CU

Jens
